src/components/snapshot/overview/ManageWorkspacesModal.tsx

```diff
@@ -104,52 +73,67 @@ export class ManageWorkspacesModal extends React.PureComponent<
   }
 
   handleClickOpen = () => {
-    this.setState({
-      open: true,
-    });
+    this.setState({ open: true });
   };
 
   handleClose = () => {
     this.setState({ open: false });
   };
 
   render() {
-    const { classes, modalText, entries, removeWorkspace, isLoading } = this.props;
+    const { modalText, entries, removeWorkspace, isLoading } = this.props;
     const { open } = this.state;
-    const button = (
-      <Button
-        className={classes.openButton}
-        aria-label={modalText}
-        onClick={this.handleClickOpen}
-        disableFocusRipple={true}
-        disableRipple={true}
-      >
-        <i className={`${classes.openButtonIcon} fa-solid fa-pen-circle`} />
-        {modalText}
-      </Button>
-    );
 
     return (
-      <span>
-        {button}
+      <Box component="span">
+        <OpenButton
+          aria-label={modalText}
+          onClick={this.handleClickOpen}
+          disableFocusRipple
+          disableRipple
+        >
+          <Box
+            component="i"
+            className="fa-solid fa-pen-circle"
+            sx={{ marginRight: '5px', top: (theme) => theme.spacing(1) }}
```

Comment:
```suggestion
            sx={{ marginRight: '5px', top: 1 }}
```
There's actually a shortcut here for spacing styles with the theme: https://mui.com/system/getting-started/the-sx-prop/#spacing